docs/static/settings-file.asciidoc

```diff
@@ -27,6 +27,22 @@ pipeline.batch.size: 125
 pipeline.batch.delay: 5
 -------------------------------------------------------------------------------------
 
+The `logstash.yml` file also supports bash-style interpolation of environment variables in
+setting values.
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+pipeline:
+  batch:
+    size: ${BATCH_SIZE}
+    delay: ${BATCH_DELAY:5}
+node:
+  name: "node_${LS_NODE_NAME}"
```

Comment:
I'd also use the default value notation in the string interpolation example so it leaves no doubt that the default feature is supported in both scenarios.